Meta 274 sip 2 0 closing the eark gap (#130)
viaacode/documentation: docs/diginstroom/sip/index.md

The meemoo SIP specification is available in the following versions:

## Version [2.0]({{ site.baseurl }}{% link docs/diginstroom/sip/2.0/index.md %}) <span class="label label-yellow">Editor's Draft</span>

> This major release fully aligns the meemoo SIP with the [EARK CSIP specification](https://earksip.dilcis.eu/).

Reviewer: ```suggestion
This major release ensures adherence of the meemoo SIP to the [EARK CSIP specification](https://earksip.dilcis.eu/).
```

### What was **added** compared to the previous version?

> Nothing

Reviewer: Perhaps something a bit more formal? 'Nothing' sounds like no real work was undertaken :D 
```suggestion
Not applicable
```
